schemas/core/modes/MODE_CAR_PARK.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Schema for MODE_CAR_PARK meta field",
+  "type": "object"
```

Comment:
Shouldn't it come with some properties?

If we do not intend to send any properties, then probably we do not need this mode at all, and if we need, then they need to be defined